server/src/api/contacts/contacts.service.ts

```diff
@@ -11,7 +11,7 @@ export const findAll = async (params: any): Promise<response> => {
             WHERE ($1::text IS NULL OR LOWER(firstname) LIKE $1)
             AND ($2::text IS NULL OR LOWER(lastname) LIKE $2)
             AND ($3::text IS NULL OR LOWER(email) LIKE $3)
-            AND ($4::text IS NULL OR LOWER(custaddress) LIKE $4)
+            AND ($4::text IS NULL OR LOWER(address) LIKE $4)
             AND ($5::text IS NULL OR LOWER(donorbadge) LIKE $5)
             AND ($6::boolean IS NULL OR seatingaccom = $6)
             AND ($7::boolean IS NULL OR vip = $7)
```

Comment:
Do you mind fixing this query in this PR? `phone` is missing from the SQL query and `seatingaccom` is missing from the values. While we're at it, we might as well change `seatingaccom` to be a boolean in the database. I am pretty certain that changing it's type wont require any frontend changes (the frontend is already handling like other boolean values)

Reply:
`seatingaccom` is already a boolean in the DB, will add `phone` to the query in my next push

Reply:
Oops I think I meant `donorbadge` can be changed to a boolean

Reply:
`donorbadge` is changed to a boolean now